.github/workflows/aes_kw.yml

```diff
@@ -0,0 +1,57 @@
+name: aes_kw
```

Comment:
I'd suggest calling it `aes-kw` to match the [crate name we have registered](https://crates.io/crates/aes-kw)